uktrade/datahub-header
# Replace node-sass with dart-sass

There is a vulnerability in a deep dependency of `node-sass`, however this package has been deprecated. I've replaced it with `dart-sass`, which was made by the same maintainers.

I've also deleted the MI template file (as the service is deprecated) and added a step to the readme that generates the required CSS files.

## Test instructions
Run `npm install`, `npm sass:dev` and `npm run start`. The header should load correctly and look as expected.
